controller/controller.go

```diff
@@ -555,7 +557,7 @@ waitForLock:
 					msg := "error when deleting old rooms. Maestro will scale up, if necessary, with previous room configuration"
 					return maestroErrors.NewKubernetesError(msg, err)
 				default:
-					err := deletePod(logger, mr, clientset, redisClient.Client, schedulerName, pod.GetName())
+					err := deletePod(logger, mr, clientset, redisClient.Client, schedulerName, gameName, pod.GetName(), "update")
```

Comment:
It is better if the reasons are at models/contants.go and you call the variables